agent/php_execute.c

```diff
@@ -491,6 +491,10 @@ static nr_library_table_t libraries[] = {
 
     {"MongoDB", NR_PSTR("mongodb/src/client.php"), nr_mongodb_enable},
 
+    /* php-amqplib RabbitMQ >= 3.7 */
```

Comment:
I don't think this comment is accurate - `phpamqplib/connection/abstractconnection.php` file also exists in version 2.x (starting with version [v2.0.2](https://github.com/php-amqplib/php-amqplib/blob/v2.0.2/PhpAmqpLib/Connection/AbstractConnection.php)) of php-amqplib.

Reply:
That's a good point.  The should say something like `PHP Agent Supports RabbitMQ >= 3.7`

They aren't maintaining 2.x and the highest version works with PHP < 8.

Preliminary testing shows it basically works ok and creates the message segments.

I updated the comment here: [127e62a](https://github.com/newrelic/newrelic-php-agent/pull/1009/commits/127e62afa3d31e4861d051567c0746357c21f5cd).